pkg/okteto/pipeline.go

```diff
@@ -404,18 +407,21 @@ func (c *OktetoClient) GetResourcesStatusFromPipeline(ctx context.Context, name
 	status := make(map[string]string)
 	for _, d := range queryStruct.Space.Deployments {
 		if string(d.DeployedBy) == name {
-			status[string(d.ID)] = string(d.Status)
+			resourceName := fmt.Sprintf("deployment/%s", d.Name)
```

Comment:
The logic for the key creation to access an element in the status map is duplicated and prone to bugs. Can we consolidate it into a function to eliminate the duplication and reduce the change of a bug occurring by an update to the key creation in either status write or read and not in the other location? Probably, something that takes a map and some constant indicating the resource type and either applies the change or returns what should be used for the key.

Reply:
A test may make sense for that function as well as it should be easy and help ensure that we have the correct debug code we want in place when the opportunity to capitalize on the debug logs occurs.